go/client/cmd_config.go

```diff
@@ -158,6 +158,18 @@ func (v *CmdConfigSet) Run() error {
 	if err != nil {
 		return err
 	}
+	// Might need to locate this in the service for GUI access
+	if v.Path == "mountdir" {
```

Comment:
This code can't be triggered from the  GUI since the GUI never calls into any code in `client/`